src/colony/modules/_net_proxied.js

```diff
@@ -0,0 +1,200 @@
+var util = require('util'),
+    events = require('events'),
+    net = require('net'),
+    tls = require('tls'),
+    streamplex = require('_streamplex');
+
+// NOTE: this list may not be exhaustive, see also https://tools.ietf.org/html/rfc5735#section-4
+var _PROXY_LOCAL = "10.0.0.0/8 172.16.0.0/12 192.168.0.0/16 169.254.0.0/16 127.0.0.0/8 localhost";
+
+var _PROXY_DBG = ('_PROXY_DBG' in process.env) || false,
+    PROXY_HOST = process.env.PROXY_HOST || "proxy.tessel.io",
+    PROXY_PORT = +process.env.PROXY_PORT || 443,
+    PROXY_TRUSTED = +process.env.PROXY_TRUSTED || 0,
+    PROXY_TOKEN = process.env.PROXY_TOKEN || process.env.TM_API_KEY,
+    PROXY_LOCAL = process.env.PROXY_LOCAL || _PROXY_LOCAL,
```

Comment:
Not particularly, the env variable as defaulted here gets processed all in once place later I don't see a compelling reason to split the `split` part of that logic away?